src/test_runner.cr

```diff
@@ -18,7 +19,7 @@ module Mint
           <script>
             class TestRunner {
               constructor (suites) {
-                this.socket = new WebSocket("ws://localhost:3001/")
+                this.socket = new WebSocket("ws://#{@flags.browser_host}:#{@flags.port}/")
```

Comment:
Why not just set it to `document.location.origin`?

Reply:
Because Docker might expose other port.

Reply:
I don't understand why we we use the `port` flag instead of the `browser_port` flag? 

My understanding is for example: in docker the server is we bound to `0.0.0.0:3000` which is exposed as `127.0.0.1:4000` then the `host` is `0.0.0.0`, `port` is `3000`, `browser_host` is `127.0.0.1` and `browser_port` is `4000`

Reply:
Ohh, you're absolutely right! A typo. I'll fix it when I'll get back to PC

Reply:
@gdotdesign done

Reply:
@Namek is it?